evm_loader/program/src/types/transaction.rs

```diff
@@ -117,6 +135,299 @@ impl rlp::Decodable for Transaction {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct AccessListTx {
+    nonce: u64,
+    gas_price: U256,
+    gas_limit: U256,
+    target: Option<Address>,
+    value: U256,
+    call_data: crate::evm::Buffer,
+    r: U256,
+    s: U256,
+    chain_id: U256,
+    recovery_id: u8,
+    access_list: Vec<(Address, Vec<StorageKey>)>,
+    rlp_len: usize,
```

Comment:
`rlp_len` should be renamed and for typed transactions it should also include the first byte.

Reply:
renamed to `byte_len`